manifests/kustomize/env/platform-agnostic-tekton-multi-user/kustomization.yaml

```diff
@@ -15,13 +15,13 @@ commonLabels:
 images:
 - name: gcr.io/ml-pipeline/api-server
   newName: quay.io/aipipeline/apiserver
-  newTag: 2.1.0-rc0
+  newTag: 2.0.5
```

Comment:
2.1.0 to 2.0.5?

Reply:
I don't think kfp will release 2.1.0 anytime soon. Just want to cut a release with all the recent enhancement we have on Tekton 0.53

Reply:
We will use this release as a benchmark on the upcoming contributions to the Argo optimization.